filesystem/src/main/java/org/frankframework/filesystem/LocalFileSystem.java

```diff
@@ -52,6 +51,14 @@ public void configure() throws ConfigurationException {
 		// No Action is required
 	}
 
+	@Override
+	public void open() throws FileSystemException {
+		super.open();
+		if (root != null && !Files.exists(Paths.get(root))) {
+			createFolder(root);
+		}
```

Comment:
Ik zou open als laaste actie verwachten, die zet namelijk een vlaggetje dat aangeeft dat deze listener actief is.